web_toggle/static/src/js/bootstrap-toggle.js

```diff
@@ -0,0 +1,171 @@
+ +function ($) {
```

Comment:
After reviewing the zip file that bootstrap gives you as a download I found a LICENCE file. I have added it next to the to the two files I use from the bootstrap-toggle library.